testsuite/integration/pom.xml

```diff
@@ -48,6 +48,8 @@
 
         <ts.elytron.cli>../../shared/enable-elytron.cli</ts.elytron.cli>
         <wildfly.dir>${project.build.directory}/wildfly</wildfly.dir>
+        
+        <version.org.wildfly.jar.plugin>3.0.0.Beta1</version.org.wildfly.jar.plugin>
```

Comment:
is there a reason to get this version here (as opposed to the top-level pom.xml)?

Reply:
No reason, I moved to top level pom file.